docs/utils/data.rst

```diff
@@ -221,3 +220,32 @@ the Internet at all.
 If you have a particular URL that is giving you trouble, you can download it
 using some other tool (e.g., ``wget``), possibly on another machine, and
 then use `~astropy.utils.data.import_file_to_cache`.
+
+Astropy Data and Clusters
+=========================
+
+Astronomical calculations often require the use of a large number of different
+processes on different machines with a shared home filesystem. This can pose
+certain complexities. In particular, if the many different processes attempt to
+download a file simultaneous this can overload a server or trigger security
+systems. The parallel access to the home directory can also trigger concurrency
+problems in the Astropy data cache, though we have tried to minimize these. We
+therefore recommend the following guidelines:
+
+ * Write a simple script that sets ``astropy.utils.iers.conf.auto_download = True``
+   and then accesses all cached resources your code will need, including source name
+   lookups and IERS tables. Run it on the head node from time to time (at least twice
```

Comment:
Why twice a month?

Reply:
The timeout defaults to 30 days, so I want to make sure there's no risk of accidentally leaving it too long.

Reply:
Maybe we can clarify this point, so that we have the knowledge to change the recommendation here if the default changes.